randovania/games/dread/json_data/Elun.txt

```diff
@@ -518,7 +519,7 @@ Extra - asset_id: collision_camera_007
   > Door to Chozo Soldier Arena
       All of the following:
           Can Slide
-          Grapple Beam or Spider Magnet or Space Jump or Walljump (Beginner) or Simple IBJ
+          Grapple Beam or Spider Magnet or Space Jump or Speed Booster or Walljump (Beginner) or Simple IBJ
```

Comment:
I see Walljump (Beginner) which means this can be made trivial. But where's the rest of the Elun changes?

Reply:
There aren't any. That's what I told you. It's pretty straight forward.

Reply:
Going through it to see if there's anything that can be added. So far just minor things that dont really affect logic

Reply:
I'll go through it again and push it after I added the Chozo requirements